Makefile

```diff
@@ -205,6 +206,11 @@ build-test-libs:
 	$(CC) -shared -fPIC -o $(TEST_LIB_DIR)/libreladyn.$(SOEXT) test/native/libs/reladyn.c
 	$(CC) -shared -fPIC $(INCLUDES) -Isrc -o $(TEST_LIB_DIR)/libjnimalloc.$(SOEXT) test/native/libs/jnimalloc.c
 
+build-test-bins:
+	@mkdir -p $(TEST_BIN_DIR)
+	gcc -o $(TEST_BIN_DIR)/malloc_plt_dyn test/native/bins/malloc_plt_dyn.c
+	gcc -o $(TEST_BIN_DIR)/native_api -Isrc -ldl test/test/c/native_api.c
```

Comment:
I thought you would rename `malloc_plt_dyn` instead :)

Anyhow, let's agree on where to store test-specific C sources. I'd prefer to keep them in a directory of a particular test (like `native_api.c` in `test/c` directory). A common directory may also do, but this should be consistent across tests.

Reply:
:) I moved `malloc_plt_dyn` to `nativemem` folder.